test/test_debugger.cpp

```diff
@@ -328,6 +328,27 @@ nl::json make_rich_inspect_variables_request(int seq, const std::string& var_nam
     return req;
 }
 
+nl::json make_exception_breakpoint_request(int seq)
+{
+    nl::json except_option = {
+        {"path", nl::json::array({
+                nl::json::object({{"names", nl::json::array({"Python Exceptions"})}})
+        })},
+        {"breakMode", "always"}
+    };
+    nl::json options = nl::json::array({except_option, except_option});
+    nl::json req = {
+        {"type", "request"},
+        {"seq", seq},
+        {"command", "setExceptionBreakpoints"},
+        {"arguments", {
+            {"filters", {"raised", "uncaught"}},
+            {"exceptionOptions", options}
+        }}
+    };
+    return req;
```

Comment:
@goanpeca This is the request you need to send to have debugpy break on exceptions.

Reply:
Thanks @JohanMabille !